src/PrestaShopBundle/Resources/views/Admin/Common/Grid/template.html.twig

```diff
@@ -0,0 +1,27 @@
+<div class="card">
```

Comment:
Why create a new template ? Can we try not to have different twig templates for the 2 presenters ?

Reply:
i'll try to adapt, but keep in mind that new presenter will return a little bit different result, so we can't expect it to work exactly the same, right?